D1673/P1673.md

```diff
@@ -6521,8 +6343,8 @@ template<class ScalingFactor,
          class Accessor>
 class scaled_accessor {
```

Comment:
I can't comment directly on the line, but should we consider making those unconditionally `noexcept` member functions conditionally `noexcept`?  For example, `access` would have `noexcept(noexcept(scaling_factor_* typename Accessor::element_type(accessor_.access(p, i))))`.